buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -563,7 +563,7 @@ class ClusterFormationTasks {
     }
 
     static Task configureInstallPluginTask(String name, Project project, Task setup, NodeInfo node, String pluginName, String prefix) {
-        final FileCollection pluginZip;
```

Comment:
why is final removed?

Reply:
I will not merge the PR until there is a GA, just wanted to start running in CI. 

Gradle bumps the Groovy version, which now complains that the final variable is changed. 
It doesn't see past branches the same way java does. One more reason to switch to java.